src/components/indpro/Header.tsx

```diff
@@ -4,12 +4,18 @@ import Lock from "../../assets/icons/Lock";
 import useConfig from "../../hooks/useConfig";
 import Link from "../atoms/Link";
 import type React from "react";
+import {useEffect, useState} from "react";
+import {useGetQueryParams} from "../../hooks/useGetQueryParams";
 
 interface Props {}
 
 const Header: React.FC<Props> = () => {
   const config = useConfig();
-
+  const queryParams = useGetQueryParams();
+  const [fullUrl, setFullUrl] = useState('');
+  useEffect(() => {
+    setFullUrl('https://pros.turbotax.intuit.com' + queryParams);
```

Comment:
[OPTIONAL] could be simplified to `https://pros.turbotax.intuit.com${queryParams}`